target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,9 @@ mediatek_setup_interfaces()
 	acer,predator-w6)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 game" eth1
 		;;
+	asus,rt-ax59u)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" wan
+		;;
```

Comment:
Can be merged with this existing block:
```
	cetron,ct3003|\
	confiabits,mt7981|\
	cudy,wr3000-v1|\
	jcg,q30-pro|\
	qihoo,360t7)
		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" wan
		;;
```

Reply:
@xavifr thanks for your updates. 

A few words about the commit message：
- Please add UART installation method too because trx file at the drive.google.com can become unreachable in future. And the commit message is a primary source of the guide if we have no wiki page.
- Please add note about MAC addresses and differences from the original Asus algorithm.

Reply:
done